app/src/main/AndroidManifest.xml

```diff
@@ -11,16 +11,19 @@
     android:roundIcon="@mipmap/ic_launcher_round"
     android:supportsRtl="true"
     android:theme="@style/OppiaTheme">
-    <activity android:name=".topic.conceptcard.testing.ConceptCardFragmentTestActivity" />
-    <activity android:name=".player.state.testing.StateFragmentTestActivity" />
+    <activity android:name="org.oppia.app.topic.conceptcard.testing.ConceptCardFragmentTestActivity" />
+    <activity android:name="org.oppia.app.player.state.testing.StateFragmentTestActivity" />
     <activity android:name=".player.exploration.ExplorationActivity" />
     <activity android:name=".topic.questionplayer.QuestionPlayerActivity" />
     <activity android:name=".topic.TopicActivity" />
     <activity android:name=".player.audio.testing.AudioFragmentTestActivity" />
-    <activity android:name=".testing.InputInteractionViewTestActivity" />
+    <activity android:name="org.oppia.app.home.HomeActivity" />
     <activity android:name=".profile.ProfileActivity" />
-    <activity android:name=".story.StoryActivity" />
-    <activity android:name=".home.HomeActivity">
+    <activity android:name=".testing.InputInteractionViewTestActivity" />
+    <activity android:name=".story.StoryActivity"/>
+    <activity
+      android:name=".splash.SplashActivity"
```

Comment:
Thanks for fixing this! I've been meaning to switch the launcher app over to Splash.